src/gpuhunt/providers/lambdalabs.py

```diff
@@ -46,12 +46,13 @@ def get(
                 instance_name=instance["name"],
                 price=instance["price_cents_per_hour"] / 100,
                 cpu=instance["specs"]["vcpus"],
-                memory=float(instance["specs"]["memory_gib"]),
+                memory=float(instance["specs"]["memory_gib"]) * 1.074,
```

Comment:
Why 1.074? Is it `(1024 / 1000) ** 3`?

Reply:
@Egor-S In Lambda, the size is in GiB, must be converted to GB.